_posts/2016-10-06-next-cio.md

```diff
@@ -42,13 +42,12 @@ We are looking for a broad and diverse pool of candidates to apply for the posit
 
 - **General management and leadership**
     - Proven ability to manage significant change in a complex enterprise environment.
+    - Experience aligning operational capacity and resources (budget, people) to business strategy.
     - Experience inspiring, leading, and managing a large cross-functional team (e.g., 100+ employees/contractors).
 - **Technical experience**
     - Undergraduate or graduate degrees in computer science or similar technical discipline.
     - Experience leading the engineering and delivery of innovative technical solutions across a range of technical domains.
-- **Financial services subject matter expertise**
-    - Experience working in financial services, particularly within consumer financial services (including regulators).
-    - Experience leading compliance, operations, and technology functions within financial services organizations (including regulators).
+    - Experience developing and implementing an enterprise – wide technology strategy .
```

Comment:
Please replace "enterprise – wide" with "enterprise-wide" and remove the space prior to the period.